crates/forge_loader/src/manifest.rs

```diff
@@ -25,24 +26,68 @@ pub struct FunctionMod<'a> {
     providers: Option<AuthProviders<'a>>,
 }
 
+// Modified 
 #[derive(Default, Debug, Clone, PartialEq, Eq, Deserialize)]
 struct ModInfo<'a> {
+    function: &'a str,
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Eq, Deserialize)]
+struct MacroMod<'a> { 
+    key: &'a str,
+    function: &'a str,
```

Comment:
```suggestion
    #[serde(borrow)]
    function: Option<&'a str>,
```
The `function` property is optional, since they can just expose the Macro's main entrypoint through a resolver: https://developer.atlassian.com/platform/forge/manifest-reference/modules/macro/#properties.